docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -443,6 +443,30 @@ paths:
           description: Group not found
           schema:
             $ref: '#/definitions/Error'
+    post:
+      tags:
+        - groups
+      summary: Add a member to a group
+      operationId: addGroupMember
+      parameters:
+        - name: id
+          in: path
+          description: Public ID of the group
+          required: true
+          type: string
+        - name: user
+          in: path
+          description: |
+            User ID of the user to add to the group.
```

Comment:
Can you please add some documentation about the format of the `userid` string here?